CoreFindView/src/au/gov/asd/tac/constellation/views/find/components/BasicFindTab.java

```diff
@@ -137,9 +136,8 @@ public BasicFindTab(final FindViewTabs parentComponent) {
             updateSelectedAttributes(getMatchingAttributeList(GraphElementType.getValue(newElement)));
         });
 
-        //Set Right click SelectionOptions contextMenu
-        final SelectOptionsExtension attributeSelectOptions = new SelectOptionsExtension(inAttributesMenu);
-        attributeSelectOptions.enablePopUp();
+        //Set Right click bulk selection ContextMenu
+        attributeFilterMultiChoiceInput.enablePopUp();
```

Comment:
wondering whether this is something that could potentially be added to the constructor (like a boolean determining whether you want it enabled or not) so that you don't have to manually call it separately?